spec/features/view_spec.rb

```diff
@@ -1,10 +1,24 @@
 require "rails_helper"
 
-RSpec.describe "Signup", type: :feature do
+RSpec.describe "Views", type: :feature do
   describe "Visit home page", type: :feature do
     it "Visits home page" do
       visit root_path
       expect(page).to have_content ("Search For Flights")
     end
   end
+
+  describe "Visit sign in", type: :feature do
+    it "Visits login page" do
+      visit  login_path
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.